provider/resources.go

```diff
@@ -269,9 +264,7 @@ func Provider() tfbridge.ProviderInfo {
 		},
 		CSharp: &tfbridge.CSharpInfo{
 			RespectSchemaVersion: true,
-			PackageReferences: map[string]string{
```

Comment:
Suspiciously python SDK didn't get changed, possibly still OK?

Reply:
I think it probably should have changed the pyproject.toml. The version ranges currently there match what was set in the schema https://github.com/pulumi/pulumi-azuread/blob/c0b30233a6c7552acad54bde5aa65e6218c6c6a1/sdk/python/pyproject.toml#L4

Reply:
Naively, I expected https://github.com/pulumi/pulumi-azuread/blob/c0b30233a6c7552acad54bde5aa65e6218c6c6a1/sdk/python/pyproject.toml#L4 to change

Reply:
Per Justin this is polyfilled as a default in Python; once we upgrade codegen this will change. So OK for now.